src/app/questions/index.js

```diff
@@ -1,11 +1,84 @@
 'use strict';
 
-Object.defineProperty(exports, "__esModule", {
-  value: true
-});
-
 //Questions
-exports.default = [
+export default [
+
```

Comment:
Maybe there a way to `require` questions from trails-generator instead of hard code them here again ?